ivy/array/general.py

```diff
@@ -237,15 +237,15 @@ def scatter_nd(
         >>> indices = ivy.array([[4], [3], [1], [7]])
         >>> updates = ivy.array([9, 10, 11, 12])
         >>> scatter = indices.scatter_nd(updates, reduction='replace', out=arr)
-        >>> print(scatter)
+        >>> scatter
```

Comment:
I think we should be printing the result for every example, so I'd suggest you revert changes involving the removal of the use of the `print` function from the PR